shared/src/test/scala/io/jdrphillips/alphabetsoup/DefaulAtomSpec.scala

```diff
@@ -0,0 +1,116 @@
+package io.typechecked
+package alphabetsoup
+
+import org.scalatest.FlatSpec
+import org.scalatest.Matchers
+import shapeless.::
+import shapeless.HNil
+import shapeless.test.illTyped
+
+class DefaultAtomSpec extends FlatSpec with Matchers {
+
+  "Mixer" should "work with a supplied default" in {
+
+  case class Source(a: Int)
```

Comment:
Indentation all weird here

Reply:
Yeah something weird happened with sublime, thought I got it all, will fix this